### PR TITLE
Feature/lowest total score, Test/lowest total score

### PR DESCRIPTION
**Pull request by WZale**
This PR adds the lowest_total_score method to stat_tracker.rb, as well as the accompanying lowest_total_score test to stat_tracker_spec.rb.

### CODE REVIEW

**Review by AStilmock — APPROVED** [review]
New method looks good!

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -56,5 +56,10 @@ def average_goals_per_game
     (goals.to_f / total_games).round(2)
   end
 
-
+  def lowest_total_score
```
Comment (Jredish11) [inline lib/stat_tracker.rb:59]: fastest hands in the west!!!

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -52,4 +52,10 @@
     end
   end
 
+  describe "#lowest_total_score" do
+    it "can calculate lowest_total_score" do
+      expect(@stat_tracker.lowest_total_score).to eq(0)
```
Comment (Jredish11) [inline spec/stat_tracker_spec.rb:57]: no one scored ??? what sports is this soccer or hockey ball

**Comment by Jredish11** [comment]
methods are just absolutely flying off the board.

**Review by Jredish11 — APPROVED** [review]

**Pull request merged**